lib/ex_aws/sqs/request.ex

```diff
@@ -0,0 +1,28 @@
+defmodule ExAws.SQS.Request do
+  @moduledoc false
+
+  def request(client, queue_name, action, params \\ []) do
+    {_, http_method} = ExAws.SQS.Impl |> ExAws.Actions.get(action)
+
+    headers = [
+      {"content-type", "application/x-www-form-urlencoded"},
+    ]
+
+    ExAws.Request.request(http_method, client.config |> url(queue_name, params), [], headers, client)
+    |> parse(client.config)
+  end
+
+  def parse({:error, result}, _), do: {:error, result}
+  def parse({:ok, body}, config) do
+    case config[:json_codec].decode(body) do
```

Comment:
Yea that was definitely just copy & paste, not planning on leaving it that way. this code is very raw, I just had an hour to kill with the AWS docs so I started building this out. it still needs a lot of work. notice the url function is just stubbed out with a comment on the endpoint.

I'm not sure what to do about XML yet.